packages/protocol/test-sol/unit/governance/validators/Validators.t.sol

```diff
@@ -430,17 +443,25 @@ contract ValidatorsTest is Test, TestConstants, Utils, ECDSAHelper {
 
       result = FixidityLib.multiply(result, base);
     }
+
     return result.unwrap();
   }
 }
 
+contract TransitionToL2AfterL1 is ValidatorsTest {
```

Comment:
Shall we remove the duplicated code from Election and push to some util file?
https://github.com/celo-org/celo-monorepo/blob/736fbc209775b6f9195816ae94dff27a13be68d0/packages/protocol/test-sol/unit/governance/voting/Election.t.sol#L178